src/Core/src/Handlers/Label/LabelHandler.cs

```diff
@@ -46,7 +46,7 @@ public partial class LabelHandler : ILabelHandler
 		{
 		};
 
-		public LabelHandler() : base(Mapper)
+		public LabelHandler() : base(Mapper, CommandMapper)
```

Comment:
This was a bug that prevented the `LabelHandler`'s commands from ever reaching the control. We never noticed because we don't have anything in there right now.

Reply:
Should we add some validation on the base handler for this ?

Reply:
Technically it is valid to not have commands - we only recently added it so an exception will be a breaking change.